docs/getting-started/installing-shoko-server.mdx

```diff
@@ -19,21 +19,35 @@ the installer and simply follow the instructions listed, it should only take a c
 
 ## Docker (Linux)
 
+Follow the instructions below for running Shoko using Docker, which is the recommended solution for Linux and NAS Systems supporting Docker.
+
 Make sure you have docker installed before you continue. For most users are installing docker from their package manager is advised, this way you will keep up with the latest updates and security updates. On the homepage of [Docker](https://docs.docker.com/install/) you can find the installation procedure for your distribution of the Docker Community Edition.
 
-First things first, download the image, and unless otherwise told, install the version tagged as `latest`.
+**NOTE:** We do <u>not</u> encourage using Docker on Windows using WSL or Hyper-V containers. It creates an unneeded abstraction layer which might cause unforeseen issues. You can do this at your own risk, as we can't support this setup.
+
+### Docker commandline
+
+First things first, pull the image, and unless otherwise told, install the version tagged as `latest`.
 
 ```sh
 $ docker pull shokoanime/server:latest
 ```
-Now, you can run the command below, after substituting the paths below (the second and third volume mount point) with path(s) leading to your library and/or import folders.
+After pulling the image, you can run the command below. Mare sure you substitute the paths under <code>-v</code> with the ones from your environment.
```

Comment:
*Mare‐>Make

Reply:
@revam this is still heavily WIP right now, moving to docusaurus. I'm just doing commits whenever I leave my computer so I won't lose anything but thx